usaspending_api/api_contracts/contracts/v2/search/spending_by_transaction.md

```diff
@@ -248,6 +248,93 @@ Returns transaction records which match the keyword and award type code filters.
 + `Subsidy Cost` (required, string, nullable)
 + `Transaction Amount` (required, string, nullable)
 
+### AdvancedFilterObject (object)
++ `keywords`: `lockheed` (required, array[string], fixed-type)
++ `award_type_codes` (required, FilterObjectAwardTypes, fixed-type)
++ `place_of_performance_scope` (optional, enum[string])
+    + Members
+        + `domestic`
+        + `foreign`
++ `place_of_performance_locations` (optional, array[LocationObject], fixed-type)
++ `agencies` (optional, array[AgencyObject], fixed-type)
++ `recipient_search_text`: [`Hampton`, `Roads`] (optional, array[string])
+    + Text searched across a recipient’s name, UEI, and DUNS
++ `recipient_scope` (optional, enum[string])
+    + Members
+        + `domestic`
+        + `foreign`
++ `recipient_locations` (optional, array[LocationObject], fixed-type)
++ `recipient_type_names`: [`category_business`, `sole_proprietorship`] (optional, array[string])
++ `award_ids`: [`SPE30018FLGFZ`, `SPE30018FLJFN`] (optional, array[string])
+    Award IDs surrounded by double quotes (e.g. `"SPE30018FLJFN"`) will perform exact matches as opposed to the default, fuzzier full text matches.  Useful for Award IDs that contain spaces or other word delimiters.
++ `award_amounts` (optional, array[AwardAmounts], fixed-type)
++ `program_numbers`: [`10.331`] (optional, array[string])
++ `naics_codes` (optional, NAICSCodeObject)
++ `tas_codes` (optional, array[TASCodeObject], fixed-type)
++ `psc_codes` (optional, enum[PSCCodeObject, array[string]])
+    Supports new PSCCodeObject or legacy array of codes.
++ `contract_pricing_type_codes`: [`J`] (optional, array[string])
++ `set_aside_type_codes`: [`NONE`] (optional, array[string])
++ `extent_competed_type_codes`: [`A`] (optional, array[string])
++ `treasury_account_components` (optional, array[TreasuryAccountComponentsObject], fixed-type)
++ `object_class` (optional, array[string])
++ `program_activity` (optional, array[number])
++ `def_codes` (optional, array[DEFC], fixed-type)
+    If the `def_codes` provided are in the COVID-19 or IIJA group, the query will only return transactions that meet two requirements:
+    1. The transaction's associated prime award has at least one File C record with one of the supplied DEFCs.
+    2. The matching DEFC's associated public law has an enactment date prior to the transaction's action_date.
+
+### TimePeriodObject (object)
```

Comment:
Does this endpoint support new awards only?

Reply:
No, I just dug into this a bit, and it only currently supports `action_date`. I'll update this contract and create a new ticket to add this functionality.